integrasjon/dokgen-klient/src/main/java/no/nav/foreldrepenger/melding/integrasjon/dokgen/dto/opphørSVP/OpphørPeriode.java

```diff
@@ -0,0 +1,100 @@
+package no.nav.foreldrepenger.melding.integrasjon.dokgen.dto.opphørSVP;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import no.nav.foreldrepenger.melding.geografisk.Språkkode;
+import no.nav.foreldrepenger.melding.integrasjon.dokgen.dto.felles.Årsak;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+import static no.nav.foreldrepenger.melding.typer.Dato.formaterDato;
+
+public class OpphørPeriode {
```

Comment:
Savner:
`@JsonAutoDetect(getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, fieldVisibility = JsonAutoDetect.Visibility.ANY)`
Uten vil muligens getterne for LocalDate-feltene gjøre at de blir serialisert med til tross for JsonIgnore? Uansett greit å gjøre det likt på alle DTOene/brevene.

Reply:
enig, takk